inc/config.php

```diff
@@ -45,6 +45,24 @@
 Session::setPath();
 Session::start();
 
+//init cache
+$GLPI_CACHE = false;
+try {
+   $cache_conf = GLPI_CACHE_CONFIG;
+   if (!isset($cache_conf['options'])
+       || !isset($cache_conf['options']['namespace'])) {
+      $cache_conf['options']['namespace'] = 'glpicache';
```

Comment:
Should probably use GLPI_VERSION to avoid bad answer in multi-glpi versions case or something related to the running instance.

Reply:
Good idea, yes; I'll add it